Dockerfile

```diff
@@ -20,8 +20,8 @@ RUN chmod a+x /autoexec.sh
 RUN mkdir /app
 WORKDIR /app
 ADD . /app
-RUN pip install -r /app/requirements.txt
-RUN python setup.py develop
+RUN pip3 install -r /app/requirements.txt
+RUN python3 setup.py develop
```

Comment:
What is the reason to update to Python 3 here?
I'm not against It, just curious.

Reply:
No other reason other than consistency, perceived increase of clarity, and force of habit.

Reply:
I'm fine with reverting this though, since in the `python:3.6-slim` base image, `python` is an alias for `python3`; and this change would somewhat break Python 2 compatibility.

Reply:
Ok than. Let's wait for couple of days. If nobody will come and disagree we can freely update It to 3.6.

Reply:
What could cause problems with this? There is only one Python version in the new base image, and `python` is an alias to `python3` in it.

_Or... are you asking why upgrade to Python3 at all, and just happened to comment on these two lines, and not the `FROM` directive?_

Reply:
At this point I have realized that there was no necessity to update python version in FROM directive and onward, so I've asked here. But thanks for pointing this out, I took a note.